tests/suites/deploy/deploy_bundles.sh

```diff
@@ -104,9 +104,9 @@ run_deploy_exported_charmhub_bundle_with_float_revisions() {
 	juju deploy ${bundle}
 
 	echo "Create telegraf_bundle_without_revisions.yaml with known latest revisions from charmhub"
-	influxdb_rev=$(juju info influxdb --format json | jq -r '."channel-map"."latest/stable".revision')
-	telegraf_rev=$(juju info telegraf --format json | jq -r '."channel-map"."latest/stable".revision')
-	ubuntu_rev=$(juju info ubuntu --format json | jq -r '."channel-map"."latest/stable".revision')
+	influxdb_rev=$(juju info influxdb --format json | jq -r '."channels"."latest"."stable"[0].revision')
+	telegraf_rev=$(juju info telegraf --format json | jq -r '."channels"."latest"."stable"[0].revision')
+	ubuntu_rev=$(juju info ubuntu --format json | jq -r '."channels"."latest"."stable"[0].revision')
```

Comment:
This diff shows the changes that are necessary to consumers of `juju info --format=json|yaml`.